docs/reference/rollup/migrating-to-downsampling.asciidoc

```diff
@@ -0,0 +1,119 @@
+[role="xpack"]
+[[rollup-migrating-to-downsampling]]
+=== {rollup-cap} Migrating to downsampling
+++++
+<titleabbrev>Migrating to downsampling</titleabbrev>
+++++
+
+Rollup and downsampling are two different features that allow historical metrics to be rolled up.
+From a high level rollup offers more functionality compared to downsampling, but downsampling is a more robust and
+easier feature to downsample metrics.
```

Comment:
I'd consider removing this entirely, because (from what I can tell) this page is really all about migrating away from a deprecated feature

Reply:
+1 I will remove the `e following features are missing from downsampling` section from this doc. In order to focus more on the migrating part.